docs/BinSkimRules.md

```diff
@@ -196,7 +196,7 @@ The BIND_NOW flag is missing from this binary, so relocation sections in '{0}' w
 
 ---
 
-## Rule `BA3030.UseGccCheckedFunctions`
+## Rule `BA3030.UseCheckedFunctionsWithGcc`
 
 ### Description
 
```

Comment:
>C# [](http://example.com/codeflow?start=93&length=2)

Does it support VB?

`Source Link is a language- and source-control agnostic system for providing first-class source debugging experiences for binaries...`